tasks/install.yml

```diff
@@ -9,13 +9,12 @@
   tags:
     - verify
 
-- assert: 
-    that: 
-      - "'java version' in java.stderr" 
+- assert:
+    that:
+      - "'java version' in java.stderr"
 
-# download artifactory http://bit.ly/Hqv9aj
-- name: 'make sure directory exists for artifactory'
-  file: dest=/opt/artifactory state=directory mode=0755
+- name: 'create user to run artifactory'
+  user: name={{artifactory_username}} home=/opt/artifactory/ shell=/bin/false system=yes
```

Comment:
It's wise to create service accounts as 'system' users. It's also wise to set their shell to /bin/false so that the user cannot directly login; this does not prevent another user (sysadmin) from running commands or shells as the artifactory user.